collect_app/src/main/java/org/odk/collect/android/activities/InstanceUploaderList.java

```diff
@@ -404,4 +447,19 @@ private boolean showSentAndUnsentChoices() {
         alertDialog.show();
         return true;
     }
+
+    @Override
+    protected void onDestroy() {
+        super.onDestroy();
+
+        if (listAdapter != null) {
+            ((InstanceUploaderAdapter) listAdapter).onDestroy();
+        }
+    }
+
+    public void setLightThemeBackground() {
+        if (!new ThemeUtils(this).isDarkTheme()) {
+            getWindow().getDecorView().setBackgroundColor(ContextCompat.getColor(this, R.color.card_ui_activity_background));
```

Comment:
Changing the background color only for the combination of this activity and light theme doesn't seem like an ideal solution. Options are to either use this background color in all activities by adding to the current theme or changing the background color of the CardView. What do you think?

Reply:
It all depends. In my opinion, activities that have card views look better with a grey background because the card view is white. The text and other elements on the card view stand out more when it's white than if it's background were to be changed so I think this approach looks fine because it fits with the colour scheme for the rest of the apps. Changing the background colour of every activity to this colour might not be ideal since it only works for certain scenarios and this approach allowed me to leave the rest of the theming alone because all  the cardview and activity ended up looking great when I switched to dark mode. I am gonna look at it some more because I know we want to remain consistent across the app in relation to theming.